identity/background.js

```diff
@@ -0,0 +1,233 @@
+
+let providers = {
+  facebook: {
+    clientId: '1094807753900920',
+    clientSecret: '3d4af261312493c4397e8003c9139500',
```

Comment:
yeah, it's no big deal.  But should link to docs I suppose.  And kill off those apps in my account.